qiita_ware/ebi.py

```diff
@@ -105,21 +86,44 @@ class EBISubmission(object):
     Parameters
     ----------
     preprocessed_data_id : str
+
+    Raises
+    ------
+    EBISumbissionError
+        - If the action is not in EBISubmission.valid_ebi_actions
```

Comment:
I added them to make them clearer while reading in text format. However, I can remove if you feel strongly about it and _you are sure!_